docs/example_python.md

```diff
@@ -1,30 +1,194 @@
 ---
-title: "Example: Python"
+title: "Example: Python + Flask"
 layout: docs
 ---
 
-You're a Python developer. You want to set up Tilt with your project.
+The best indicator of a healthy development workflow is a short feedback loop.
 
-We hear you! We're working on a better guide for this.
+Kubernetes is a huge wrench in the works.
 
-For now, check out:
+Let's fix this.
 
-- [abc123](https://github.com/windmilleng/abc123) a mini microservice app with a Python server called `numbers`.
-- Servantes (below), our microservice app in multiple languages.
+In this example, we're going to take you through a simple “hello world” server written in Python that uses [Flask](https://palletsprojects.com/p/flask/), a lightweight web application framework.  (Want this guide for Django or another framework? [Let us know](https://tilt.dev/contact).)
 
-## Servantes
+We'll use Tilt to:
 
-[Servantes](https://github.com/windmilleng/servantes) is a personalized homepage
-app implemented with many microservices in multiple languages. You can run it
-with the commands
+- Run the server on Kubernetes
+- Measure the time from a code change to a new process
+- Optimize that time for fast feedback
 
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
+as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-python](https://github.com/windmilleng/tilt-example-python)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-python/tree/master/2-live_update)
+
+## Step 0: The Simplest Deployment
+
+Our server is just seven lines long, and all it does is serve us an HTML page:
+
+```python
+from flask import Flask, render_template
+app = Flask(__name__)
+
+
+@app.route("/")
+def serve():
+    return render_template("index.html")
+
+
+if __name__ == "__main__":
+    app.run()
 ```
-git clone https://github.com/windmilleng/servantes
-cd servantes; tilt up
+
+To start this server on Kubernetes, we need 3 configs:
+
+1. A [Dockerfile](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/Dockerfile) that builds the image
+2. A [Kubernetes deployment](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/kubernetes.yaml) that runs the image
+3. And finally, a [Tiltfile](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/Tiltfile) that ties them together:
+
+```python
+docker_build('example-python-image', '.')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-python', port_forwards=5000)
```

Comment:
can we use the same application port for all our examples?

We usually recommend people use port 5000 for the registry, see:
https://docs.docker.com/registry/deploying/
https://github.com/windmilleng/kind-local/blob/master/kind-with-registry.sh

and this example will not work for them as-is

Reply:
Can confirm, just ran into this.